retwist/param_resource.py

```diff
@@ -37,7 +37,7 @@ def parse_args(self, request):
             try:
                 val = param.parse_from_request(name, request)
             except Error as ex:
-                error_msg = "Error in parameter {}: {}".format(name, ex.message).encode("utf-8")
+                error_msg = "Error in parameter {}: {}".format(name, ex.message.decode()).encode("utf-8")
```

Comment:
Maybe it's simpler to just do:

    error_mgs = b"Error in parameter %s: %s" % (name, ex.message)

… or similar, to cut down on the number of de/encodes here.

Reply:
Refactored.